core/builder/src/main/java/io/quarkus/builder/BuildStepBuilder.java

```diff
@@ -185,6 +192,22 @@ public BuildStepBuilder consumes(Class<? extends BuildItem> type, ConsumeFlags f
         return this;
     }
 
+    public BuildStepBuilder ordinal(int ordinal) {
```

Comment:
We wouldn't want this to be public.  The ordinal could possibly be automatically set using a global counter on `BuildChainBuilder`.

Reply:
Good catch. The method and field are actually unused since the last iteration, so they can be removed.